specification/communication/data-plane/TrunkStatus/preview/2022-07-01-preview1/communicationservicestrunkstatus.json

```diff
@@ -0,0 +1,212 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a list of all Trunks health configured for resource.": {
+            "$ref": "./examples/GetTrunksHealths.json"
+          }
+        }
+      }
+    },
+    "/sip/trunksHealth/{fqdn}": {
+      "get": {
+        "tags": [
+          "TrunkHealth"
+        ],
+        "description": "This endpoint gets single Trunk health configured for resource.",
+        "summary": "Get single Trunk health configured for resource.",
+        "operationId": "GetTrunkHealth",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "path",
+            "name": "fqdn",
+            "description": "Case insensitive FQDN of the trunk.",
+            "required": true,
+            "type": "string",
+            "maxLength": 256
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok.",
+            "schema": {
+              "$ref": "#/definitions/TrunkHealth"
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets Trunk health.": {
+            "$ref": "./examples/GetTrunksHealth.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ListOfTrunksHealth": {
+      "description": "Health of SIP trunks.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Represents an array of SIP trunks health.",
+          "maxLength": 250,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TrunkHealth"
+          }
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "TrunkHealth": {
+      "description": "Represents health of a SIP trunk for routing calls.",
+      "type": "object",
+      "required": [
+        "fqdn",
+        "tls",
+        "ping",
+        "overallStatus"
+      ],
+      "properties": {
+        "fqdn": {
+          "description": "Trunk's FQDN of the paired SBC",
+          "maxLength": 256,
+          "type": "string"
+        },
+        "tls": {
+          "description": "The status of the TLS connections between Direct Routing and the SBC",
+          "type": "string",
+          "enum": [
+            "NoData",
+            "OK",
+            "CertExpiring",
+            "CertExpired"
+          ],
+          "x-ms-enum": {
+            "name": "TlsStatusType",
+            "modelAsString": true
+          }
+        },
+        "ping": {
+          "description": "The status of options message send by SBC",
+          "type": "string",
+          "enum": [
+            "NoData",
```

Comment:
`NoData` is an unconventional default value for an enum.

A quick search suggests that `unknown` is a more conventional choice (also in my experience):

https://github.com/Azure/azure-rest-api-specs/search?q=Unknown
https://github.com/Azure/azure-rest-api-specs/search?q=Undetermined
https://github.com/Azure/azure-rest-api-specs/search?q=NoData